### ISSUE TITLE
KeyUp & KeyDown not working

### ISSUE THREAD

**Issue by beli3ver** [ihabunek/toot#30]
Hi there,

the KeyUp & KeyDown option under Debian is not working

https://github.com/ihabunek/toot/blob/master/toot/app.py#L79

Thanks

**Comment by ihabunek** [comment]
@beli3ver This should be fixed by PR #39, if you have some time check out the code and let me know if it works.

**Comment by beli3ver** [comment]
I get this error:

```bash
malte@lin:~$ toot curses
Traceback (most recent call last):
  File "/usr/bin/toot", line 11, in <module>
    load_entry_point('toot==0.16.2', 'console_scripts', 'toot')()
  File "/usr/lib/python3/dist-packages/toot/console.py", line 312, in main
    run_command(app, user, command_name, args)
  File "/usr/lib/python3/dist-packages/toot/console.py", line 288, in run_command
    return fn(app, user, parsed_args)
  File "/usr/lib/python3/dist-packages/toot/commands.py", line 69, in curses
    TimelineApp(generator).run()
  File "/usr/lib/python3/dist-packages/toot/app.py", line 44, in run
    curses.wrapper(self._wrapped_run)
  File "/usr/lib/python3.6/curses/__init__.py", line 94, in wrapper
    return func(stdscr, *args, **kwds)
  File "/usr/lib/python3/dist-packages/toot/app.py", line 61, in _wrapped_run
    self.full_redraw()
  File "/usr/lib/python3/dist-packages/toot/app.py", line 138, in full_redraw
    self.draw_statuses(self.left)
  File "/usr/lib/python3/dist-packages/toot/app.py", line 185, in draw_statuses
    self.draw_status_row(window, status, offset, highlight)
  File "/usr/lib/python3/dist-packages/toot/app.py", line 177, in draw_status_row
    window.addstr(offset + 4, 1, '─' * (width - 2))
_curses.error: addwstr() returned ERR
```

**Comment by ihabunek** [comment]
@beli3ver can you please upgrade to v.0.17.0 which was just released and try again? If you still get the error, please tell me which OS and terminal you're using.

**Comment by beli3ver** [comment]
```bash
➜ toot curses       
Traceback (most recent call last):
  File "/usr/bin/toot", line 11, in <module>
    load_entry_point('toot==0.17.0', 'console_scripts', 'toot')()
  File "/usr/lib/python3/dist-packages/toot/console.py", line 338, in main
    run_command(app, user, command_name, args)
  File "/usr/lib/python3/dist-packages/toot/console.py", line 314, in run_command
    return fn(app, user, parsed_args)
  File "/usr/lib/python3/dist-packages/toot/commands.py", line 67, in curses
    from toot.ui.app import TimelineApp
ModuleNotFoundError: No module named 'toot.ui'
```
> Linux lin 4.14.0-3-amd64 #1 SMP Debian 4.14.12-2 (2018-01-06) x86_64 GNU/Linux

> rxvt-unicode (urxvt) v9.22 - released: 2016-01-23

**Comment by ihabunek** [comment]
@beli3ver That was a packaging issue (#41), fixed it and released `0.17.1`. Please upgrade and try again.

**Comment by beli3ver** [comment]
Ok, thanks it works.
KeyUp and KeyDown working now
